Source/WTF/wtf/PlatformUse.h

```diff
@@ -348,8 +348,7 @@
 #endif
 
 #if !defined(USE_LIBPAS_JIT_HEAP) && !USE(SYSTEM_MALLOC)
-#include <bmalloc/BPlatform.h>
-#if BENABLE(LIBPAS) && OS(DARWIN)
+#if defined BENABLE_LIBPAS && BENABLE_LIBPAS && OS(DARWIN)
```

Comment:
`BENABLE_LIBPAS` also comes from `bmalloc`, so this change just make the condition always false.

Reply:
Makes sense. In https://github.com/WebKit/WebKit/commit/92059206b60d7b87624c4f002cc58d3601ac0d84, it's inlined further.